Face/FindSimilar/Program.cs

```diff
@@ -0,0 +1,198 @@
+            await Client.FaceList.CreateAsync(faceListId, faceListId, "face list for sample");
+
+            foreach (var targetImageFileName in targetImageFileNames)
+            {
+                // Read image file.
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open))
+                {
+                    // Add face to face list.
+                    Console.WriteLine($"add faces to face list from image `{targetImageFileName}`.");
+                    var faces = Client.FaceList.AddFaceFromStreamAsync(faceListId, stream, targetImageFileName).Result;
+                    if (faces == null)
+                    {
+                        Console.WriteLine($"[Error] No persisted face from image `{targetImageFileName}`.");
+                        return;
+                    }
+                }
+            }
+
+            // Get persisted faces.
+            List<PersistedFace> persistedFaces = Client.FaceList.GetAsync(faceListId).Result.PersistedFaces as List<PersistedFace>;
+
+            // Read image file.
+            using (FileStream stream = new FileStream(Path.Combine("Images", sourceImageFileName), FileMode.Open))
+            {
+                // Detect faces from image stream.
+                IList<DetectedFace> detectedFaces = Client.Face.DetectWithStreamAsync(stream).Result;
+                if (detectedFaces == null || detectedFaces.Count == 0)
+                {
+                    Console.WriteLine($"[Error] No face detected from image `{sourceImageFileName}`.");
+                    return;
+                }
+
+                Console.WriteLine($"{detectedFaces.Count} faces detected from image `{sourceImageFileName}`.");
+                if (detectedFaces[0].FaceId == null)
+                {
+                    Console.WriteLine("[Error] Parameter `returnFaceId` of `DetectWithStreamAsync` must be set to `true` (by default) for verification purpose.");
+                    return;
+                }
+
+                // Find similar example for find similar faces from the face list with similar confidence.
+                var similarResults = Client.Face.FindSimilarAsync(detectedFaces[0].FaceId.Value, faceListId).Result;
+                foreach (var similarResult in similarResults)
+                {
+                    Console.WriteLine($"Faces from {sourceImageFileName} & {persistedFaces.Find(p => p.PersistedFaceId == similarResult.PersistedFaceId).UserData} are of the same (Positive) person, similarity confidence: {similarResult.Confidence}.");
+                }
+            }
+
+            // Delete the face list.
+            await Client.FaceList.DeleteAsync(faceListId);
+            Console.WriteLine($"{Environment.NewLine}Delete FaceList {faceListId}");
+
+            Console.WriteLine("\nPress ENTER to exit.");
+        }
+
+        public static async void FindSimilar_in_LargeFaceList()
```

Comment:
`public static async Task FindSimilar_in_LargeFaceList()`

Reply:
Same for the 3 functions.

Reply:
done